### PR TITLE
Re-submit of earlier PR combined into one commit

### PR DESCRIPTION
**Pull request by SkorpionNL**
- Version bump to 0.6.0
- Combine filters for Mi button, Aqara button & Symfonisk remote for 1-4x button presses in app.json.
- Add check for reachable state for sensors.
- Added On/Off plug-in unit as another Smart Plug type
- New Smart plug Icon
- Fixed some typo's
- Added support for:
Aqara Wireless Button (WXKG11LM)
Ikea Symfonisk Sound Controller

### CODE REVIEW

**Comment by SkorpionNL** [comment]
Some things went wrong when I installed GitHub Desktop today, so I re-made my master branch and re-submitted my PR including the changes you requested on my previous PR.

File: app.json [hunk 1]
```diff
@@ -619,58 +675,90 @@
...
+            "filter": "driver_id=mi-button|aqara-button"
           }
         ]
       },
       {
-        "id": "button_held",
+        "id": "start_ccw_rotate",
         "title": {
-          "en": "Button held"
+          "en": "Button has been rotated counter clockwise",
```
Comment (notabene00) [inline app.json]: Can you please change "button" to "device" or something (with translation to nl)
So I can use this trigger in tradfri wireless (round) dimmer implementation, which is not "button"

File: app.json [hunk 2]
```diff
@@ -619,58 +675,90 @@
...
+            "filter": "driver_id=symfonisk-remote"
           }
         ]
       },
       {
-        "id": "button_released",
+        "id": "start_cw_rotate",
         "title": {
-          "en": "Button released"
+          "en": "Button has been rotated clockwise",
```
Comment (notabene00) [inline app.json]: Same here

File: app.json (hunk 1 above)
Comment (SkorpionNL) [reply app.json]: Is "remote" ok?

Comment (notabene00) [reply app.json]: Yes

Comment (SkorpionNL) [reply app.json]: Done

**Comment by notabene00** [comment]
Great! Thanks!

**Pull request merged**